<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomWindow</class>
 <widget class="QDialog" name="CustomWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>359</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Удаление игры</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit, QPushButton {
    border-radius: 15px;
    padding: 10px;
    font-size: 14px;
}

QDialog {
    border-radius: 10px;
}
QLabel {
    font-size: 12px;
}
QLabel#gameNameLabel {
    color: #4CAF50;
    font-size: 14px;
    font-weight: bold;
    padding: 8px;
    border-radius: 5px;
}
QPushButton {
    background-color: #4CAF50;
    color: white;
    border: none;
    border-radius: 5px;
    padding: 8px 15px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #45a049;
}
QPushButton:pressed {
    background-color: #3d8b40;
}
QPushButton#acceptButton {
    background-color: #f44336;
}
QPushButton#acceptButton:hover {
    background-color: #da190b;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="gameNameLabel">
     <property name="text">
      <string>Удалить игру {Игра}?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="placeholderText">
      <string>Имя</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="oldPasswordLineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Старый пароль</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="newPasswordLineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Новый пароль</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="confirmNewPasswordLineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Подтверждение пароля</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="acceptButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="contunieButton">
     <property name="text">
      <string>Продолжить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
